docs/standard/using-linq.md

```diff
@@ -69,7 +69,7 @@ For a significant chunk of software out in the wild, everything revolves around
 Consider the following: finding all XML elements with a specific attribute value.
 
 ```cs
-public static FindAllElementsWithAttribute(XElement documentRoot, string elementName,
+public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                            string attributeName, string value)
```

Comment:
This breaks alignment for the second line of parameters. Compare original:

```c#
public static FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                           string attributeName, string value)
```

Modified:

```c#
public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                           string attributeName, string value)
```

You should align the second line like this:

```c#
public static IEnumerable<XElement> FindAllElementsWithAttribute(XElement documentRoot, string elementName,
                                                                 string attributeName, string value)
```

Also, I think this is too wide, so it will appear with a scrollbar on docs.ms. Maybe some other formatting would be better?

Reply:
bb15293c569fa92f1acf6315c5fe7f2cd9cb6f68